README.md

```diff
@@ -61,4 +61,16 @@ To generate an installable extension file:
 web-ext build
 ```
 
-The output will be a `.zip` file in `web-ext-artifacts/` - this should be renamed to `.xpi` for Firefox and `.crx` for Chrome.
\ No newline at end of file
+The output will be a `.zip` file in `web-ext-artifacts/` - this should be renamed to `.xpi` for Firefox and `.crx` for Chrome.
+
+## Building and testing a study locally
+
+To run an end-to-end local test with your own study add-on, first build your study (if you don't have one, you can [build the Ion Basic Study](https://github.com/mozilla-ion/ion-basic-study)) and export the build as `<name-of-study>.xpi`. Edit `/public/locally-available-studies.json` so that `sourceURI.spec` is `/public/<name-of-study>.xpi` (you can change the other fields in `/public/locally-available-studies.json` as well for demo purposes as needed). 
```

Comment:
heh, I just discovered that you cannot in fact do this very easily :)

Reply:
The part that doesn't "work" is making an xpi. I just haven't doen it before, so my instructions were not quite accurate.